mt-kahypar/io/command_line_options.cpp

```diff
@@ -106,7 +106,13 @@ namespace mt_kahypar {
              "Summarize results in CSV format")
             ("algorithm-name",
              po::value<std::string>(&context.algorithm_name)->value_name("<std::string>")->default_value("MT-KaHyPar"),
-             "An algorithm name to print into the summarized output (csv or sqlplottools). ");
+             "An algorithm name to print into the summarized output (csv or sqlplottools). ")
+            ("use-individual-part-weights",
```

Comment:
I think we do not have to expose this parameter to the user. If --part-weights is specified, then set use_individual_part_weights to true. Should be possible with a lambda function (see e.g. --c-type).

Reply:
Yeah, I just copied the interface from KaHyPar but restricting to --part-weights probably makes more sense.